addons/web_editor/static/src/js/widgets/ace.js

```diff
@@ -360,7 +360,7 @@ var ViewEditor = Widget.extend({
         this.aceEditor.setSession(editingSession);
 
         if (this.currentType === 'xml') {
-            this.$viewID.text(_.str.sprintf(_t("Template ID: %s"), this.views[resID].xml_id));
+            this.$viewID.text(_.str.sprintf(_t("Template ID: %s"), this.views[resID].xml_id || this.views[resID].key));
```

Comment:
Should not it be 'key or xml_id' instead of 'xml_id or key'?

Reply:
Should not it be 'key' only ?

by default key = xml_id!
why try to handle the specific case of "spcific view" instead to use common field key

Reply:
IMHO all 3 solutions will have the same results as if there is a key, xml_id should be the same.
I will only use key then like in most of the website code, since it is now required and it is a 'valeur sure' :yum: .